.github/workflows/clang-format-checker.yml

```diff
@@ -17,22 +17,22 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
-      - name: install clang-format
-        run: sudo apt install -y clang-format
       - name: Comparing PR against target branch
         env:
           PR_NUMBER: ${{ github.event.number }}
           GH_TOKEN: ${{ github.token }}
         run: |
-            echo Comparing $GITHUB_BASE_REF vs HEAD
-            git diff -U0 --no-color origin/$GITHUB_BASE_REF..HEAD | clang-format-diff -p1 | tee format.diff
-            if [ -s "format.diff" ]
+            echo Comparing $PR_NUMBER vs target branch
+            git fetch origin refs/pull/$PR_NUMBER/head:pull/$PR_NUMBER
+            git checkout pull/$PR_NUMBER
+            git diff -U0 --no-color $GITHUB_REF..pull/$PR_NUMBER | clang-format-diff-15 -p1 | tee format.diff
+            if [ -s format.diff ]
             then
               echo PR contains clang-format violations. First 50 lines of the diff: >> message.txt
-              echo ``` >> message.txt
+              echo \`\`\`diff >> message.txt
               cat format.diff | head -n 50 >> message.txt
-              echo ``` >> message.txt
-              echo See [action log]\(https://github.com/microsoft/DirectXShaderCompiler/actions/runs/$GITHUB_RUN_ID/job/$GITHUB_JOB\) for the full diff.  >> message.txt
+              echo \`\`\` >> message.txt
+              echo See [action log]\(https://github.com/microsoft/DirectXShaderCompiler/actions/runs/$GITHUB_RUN_ID/) for the full diff.  >> message.txt
```

Comment:
The link in the sample provided does not work, is that expected?

See [action log](https://github.com/microsoft/DirectXShaderCompiler/actions/runs/6256158344/job/check-pr-formatting) for the full diff.

Reply:
Oh yes, because that URL points to DXC not my fork where the actual log is.